browser/brave_wallet/brave_wallet_provider_impl_unittest.cc

```diff
@@ -1222,16 +1225,20 @@ TEST_F(BraveWalletProviderImplUnitTest, SignTypedMessage) {
   mojom::ProviderError error;
   std::string error_message;
   base::Value domain(base::Value::Type::DICTIONARY);
+  std::string domain_hash = "test_domain_hash";
+  std::string primary_hash = "test_primary_hash";
```

Comment:
This is one of the reason that we need to do
https://github.com/brave/brave-core/pull/12265#discussion_r808548151
The API is not supposed to work if it takes in garbage data like this

Reply:
done